image.yaml

```diff
@@ -21,5 +21,6 @@
       dest: /
   run:
     workdir: /
+    user: 65532
```

Comment:
why add this?

Reply:
For security point of view, operator-sdk has changes the container `securityContext` to `runAsNonRoot: true`. We have a to define a non root user to run operator container.